diagnostic.md

```diff
@@ -94,4 +94,5 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+'exercise' 'name['given']['surname']' 'Jason' 'Weeks'
```

Comment:
> On Jul 11, 2016, at 11:29 AM, Jason Weeks <notifications@github.com> wrote:
> 
> In diagnostic.md <https://github.com/ga-wdi-boston/fundamentals-diagnostic/pull/67#discussion_r70280333>:
> 
> > @@ -94,4 +94,5 @@ var teamMember = {
> >  What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
> >  `teamMember['name']['surname']`?
> >  
> > -_Replace this text with your answer._
> > +'exercise' 'name['given']['surname']' 'Jason' 'Weeks'
> I'm note really sure how to read this. It looks both half right and wrong
> 
> —
> 

teamMember[‘hobby’]= ‘exercise’
teamMember[‘name’]= ‘name[‘given’][‘surname’]’
teamMember[‘name’][‘surname’]= ‘Jason’ ‘Weeks'
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/ga-wdi-boston/fundamentals-diagnostic/pull/67/files/7f0e3239ec3ec5ae075eb0e3dd1f38428392223d#r70280333>, or mute the thread <https://github.com/notifications/unsubscribe/ARzm6JdbRMGSWegJzAPP-Z83craiJ3AEks5qUmFlgaJpZM4JJZMx>.
>